peers/app_request_network.go

```diff
@@ -222,14 +222,31 @@ func (c *ConnectedCanonicalValidators) GetValidator(nodeID ids.NodeID) (*warp.Va
 
 // ConnectToCanonicalValidators connects to the canonical validators of the given subnet and returns the connected
 // validator information
-func (n *appRequestNetwork) ConnectToCanonicalValidators(subnetID ids.ID) (*ConnectedCanonicalValidators, error) {
-	// Get the subnet's current canonical validator set
+func (n *appRequestNetwork) ConnectToCanonicalValidators(
+	height uint64,
+	subnetID ids.ID,
+) (*ConnectedCanonicalValidators, error) {
+	var err error
```

Comment:
no need to set a new variable here instead of usual `err :=` assignment, will only pass line 231 if err = nil.

Reply:
I'm overriding the passed in height if it's 0 so I do need the `err` defined prior to that so that line 232 is pure assignment to already declared vars. 

I could declare a new variable to replace `height` instead but I think this is cleaner

Also hi 😄